i18n/dnd5e/gl/dnd5e.json

```diff
@@ -0,0 +1,758 @@
+{
+  "ACTOR.TypeCharacter": "Personaxe Xogador",
+  "ACTOR.TypeNpc": "Personaxe Non Xogador",
+  "ACTOR.TypeVehicle": "Vehículo",
+  "ITEM.TypeBackpack": "Mochila",
+  "ITEM.TypeClass": "Clase",
+  "ITEM.TypeConsumable": "Consumíbel",
+  "ITEM.TypeEquipment": "Equipamento",
+  "ITEM.TypeFeat": "Característica",
+  "ITEM.TypeLoot": "Botín",
+  "ITEM.TypeSpell": "Maxia",
+  "ITEM.TypeTool": "Ferramenta",
+  "ITEM.TypeWeapon": "Arma",
+  "DND5E.title": "Sistema de Xogo DnD5e",
+  "DND5E.AbbreviationCR": "ND",
+  "DND5E.AbbreviationConc": "Conc.",
+  "DND5E.AbbreviationDC": "CD",
+  "DND5E.AbbreviationLR": "DL",
+  "DND5E.AbbreviationLevel": "Nv.",
+  "DND5E.AbbreviationLbs": "lbs.",
+  "DND5E.AbbreviationSR": "DC",
+  "DND5E.Ability": "Atributo",
+  "DND5E.AbilityStr": "Forza",
+  "DND5E.AbilityStrAbbr": "for",
+  "DND5E.AbilityCon": "Constitución",
+  "DND5E.AbilityConAbbr": "con",
+  "DND5E.AbilityDex": "Destreza",
+  "DND5E.AbilityDexAbbr": "des",
+  "DND5E.AbilityInt": "Intelixencia",
+  "DND5E.AbilityIntAbbr": "int",
+  "DND5E.AbilityWis": "Sabedoria",
+  "DND5E.AbilityWisAbbr": "sab",
+  "DND5E.AbilityCha": "Carisma",
+  "DND5E.AbilityChaAbbr": "car",
+  "DND5E.AbilityModifier": "Modificador de Atributo",
```

Comment:
En galego, ao contrario do inglés ou o alemán, apenas os nomes propios van en maiúscula. Entón mellor "Modificador de atributo" (ou, máis abaixo, "Proba de atributo", etc). No ficheiro hai varias liñas con casos así 🙂